web/src/pages/profile.tsx

```diff
@@ -293,6 +295,7 @@ const ProfilePage = (props: Props): ReactElement => {
     info: (
       <>
         <ProfileTabHeader tab="info" />
+        <ProfileErrorAlert fieldErrors={getFieldErrors()} />
         {displayOpportunityAlert && <ProfileOpportunitiesAlert />}
```

Comment:
@aloverso I have a question related to the comment immediately above and your response it - `I disagree, I'd rather see this as a guard clause`

If want to add guard clauses within the component, then shouldn't examples like `displayOpportunityAlert` be embedded within the component as well? That logic currently resides at the parent level (profile page).

``` 
const displayOpportunityAlert = LookupOperatingPhaseById(
    profileData.operatingPhase
  ).displayProfileOpportunityAlert;
...
{displayOpportunityAlert && <ProfileOpportunitiesAlert />}
```

In Soma's example we are passing the errors as a prop. The pattern above is one we use often. I'm down to change the approach, it's confusing to me because it seems inconsistent.

Profile Page
`<ProfileErrorAlert fieldErrors={getFieldErrors()} />`

Component
```
interface Props {
  fieldErrors: string[];
}

export const ProfileErrorAlert = (props: Props): ReactElement => {
...
const displayProfileErrorAlert = (): boolean => props.fieldErrors.length > 0;
...
     {displayProfileErrorAlert() && (
        <>
          <Alert variant="error" dataTestid={"profile-error-alert"}>
            <div>{profileErrorAlertText()}</div>
            <ul>
              {errorFieldsIds().map((id) => (
                <li key={id}>{getHeader(id)}</li>
              ))}
            </ul>
          </Alert>
        </>
      )}
...
```